test/rbac.js

```diff
@@ -109,16 +109,18 @@ describe('rbac', () => {
       rbac._roles['role'] = new Role();
```

Comment:
we should add additional tests for the new functionality.

tests for:
- user has multiple roles and none have access to action
- user has multiple roles and some have access to action